.github/workflows/integration.yml

```diff
@@ -43,7 +43,11 @@ jobs:
       - name: Client tests
         run: cd /tmp/DIRACRepo && ./integration_tests.py test-client || touch client-tests-failed
       - name: diracx logs
+        if: ${{ failure() }}
         run: docker logs diracx
+      - name: DIRAC logs
+        if: ${{ failure() }}
+        run: cd /tmp/DIRACRepo && ./integration_tests.py logs --no-follow --lines 1000
```

Comment:
This makes things much nicer for debugging: https://github.com/DIRACGrid/diracx/actions/runs/6328508346/job/17186827358